content/docs/command-reference/repro.md

```diff
@@ -25,9 +25,10 @@ graph (a [DAG](https://en.wikipedia.org/wiki/Directed_acyclic_graph)) implicitly
 defined by the stages listed in `dvc.yaml`. The commands defined in these stages
 are then executed in the correct order.
 
-For stages with multiple commands (having a list in the `cmd` field), commands
-are run one after the other in the order they are defined. The failure of any
-command will halt the remaining stage execution, and raises an error.
+For stages with multiple commands (having a list or a multiline string in the
```

Comment:
is there a standard for "multiline strings" in YAML (starts with | or > or any string with newlines)?

Reply:
It's the one with `|`. The `>` is quite opposite, changes a newline into a space, i.e. multiline to a single line string.

```yaml
stages:
  stage1:
    cmd: |
       wget https://code.dvc.org/get-started/code.zip
       unzip code.zip
       rm -f code.zip 
```

Reply:
Thanks @skshetry ! That's exactly what I was looking for - that link. We can make the `multiline string` a link? (and probably the list also). But even in that case I would include explicitly that we mean `|` style blocks, right here in the text.

Reply:
I think we can also document this in https://dvc.org/doc/user-guide/project-structure/pipelines-files

Reply:
> haven't found a good place to accomodate this as an example, we don't have examples on dvc.yaml guide

There are many snippet examples all over. https://dvc.org/doc/user-guide/project-structure/pipelines-files#templating has an anti-multiline one for example. We should change that one if we recommend `|` better:

![image](https://user-images.githubusercontent.com/1477535/129834250-ed49a3c8-069a-4b6d-89b7-61dc0de586eb.png)

Reply:
p.s. Agree on linking and adding the info to the pipelines file guide.

Reply:
We do have examples, but not for the possibilities of every keyword, just for the high-level features.

Reply:
> Would have been great if it were collapsible

Reply:
> Would have been great if it were collapsible

Good idea! It can still be great, feel free to contribute